odoo/tools/safe_eval.py

```diff
@@ -465,6 +465,9 @@ def __repr__(self):
 mods = ['parser', 'relativedelta', 'rrule', 'tz']
 for mod in mods:
     __import__('dateutil.%s' % mod)
+# make sure to patch pytz before exposing
+from odoo._monkeypatches import monkeypatches_pytz  # noqa: E402, F401
+
```

Comment:
why is it needed? isn't it patched already?

also you forgot to call the patch() function